examples/todo-backend-typescript/run.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 export FEATUREHUB_ACCEPT_BAGGAGE=true
-export FEATUREHUB_EDGE_URL=http://localhost:8085/
-export FEATUREHUB_CLIENT_API_KEY=default/6a3ec2bc-e061-4d52-a4d2-0f80e1a40543/hnfsr2tGKDXmZx58Y1LIw8MeblAObr*fMFq4YJ3xj9aKVSGEk7h
+export FEATUREHUB_CLIENT_API_KEY=default/bf05d5c7-b494-4df1-a0f2-7a66967b7bcf/7tNxYCY7VB10gp5SzJLQwzzlNHTdRw*3AjdbcU1zI8OZebj1DIn
+export FEATUREHUB_EDGE_URL=http://localhost:8085
+export FEATUREHUB_BASE_URL=http://localhost:8085
```

Comment:
edge and base the same?

Reply:
shouldn't have been included. It is the  same here because its party-server.